michael-rubinstein/hw_02/link_to_hubot_code.txt

```diff
@@ -0,0 +1 @@
+https://github.com/hyloguy/hylohubot
```

Comment:
You two added a readme!! 😸 
Documentation earns points any day!

I'm unable to comment on specific lines of your repo so I'll do it here:
In `slackbot.js`, line 9 contains a variable with a hard coded image url. In programming best practices, we do our best to not hard code. Code needs to be dynamic, to change on a whim. 

Our Slack organization has a Giphy addon, which you can use to post semi-random gifs to the channel by including a tag with the giphy command. Say, `/giphy birthday` to have a random birthday gif posted. What do you think of having your bot use `/giphy {greeting here}` in response to a greeting?

Line 26 made me smile - there's some dynamic code! Nicely done. 👍